### PR TITLE
[DFS] Add a solution to Course Schedule & Course Schedule II

### PR DESCRIPTION
**Pull request by demonkoo**
Please have a look at my accepted solution, thank you

### CODE REVIEW

File: DFS/CourseSchedule.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * 3) Create a set of all sources
+ * 4) While the set isn’t empty,
+ *     i. Remove a vertex from the set and add it to the sorted list
+ *     ii. For every edge from that vertex:
+ *         - Decrement the in-degree of the destination node
+ *         - Check all of its destination vertices and add them to the set if they have no incoming edges
+ * Time Complexity: O(|E| + |V|), Space Complexity: O(n^2)
+ * Recommand Reading: http://cs.brown.edu/courses/csci0160/lectures/14.pdf
+ */
+class Solution {
```
Comment (soapyigu) [inline DFS/CourseSchedule.swift]: Could you please change the class name to file name?

File: DFS/CourseScheduleII.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ * 3) Create a set of all sources
+ * 4) While the set isn’t empty,
+ *     i. Remove a vertex from the set and add it to the sorted list
+ *     ii. For every edge from that vertex:
+ *         - Decrement the in-degree of the destination node
+ *         - Check all of its destination vertices and add them to the set if they have no incoming edges
+ * Time Complexity: O(|E| + |V|), Space Complexity: O(n^2)
+ * Recommand Reading: http://cs.brown.edu/courses/csci0160/lectures/14.pdf
+ */
+class Solution {
```
Comment (soapyigu) [inline DFS/CourseScheduleII.swift]: Could you please change class name to file name?

File: DFS/CourseScheduleII.swift [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ * 2) Decorate each vertex with its in-degree
+ * 3) Create a set of all sources
+ * 4) While the set isn’t empty,
+ *     i. Remove a vertex from the set and add it to the sorted list
+ *     ii. For every edge from that vertex:
+ *         - Decrement the in-degree of the destination node
+ *         - Check all of its destination vertices and add them to the set if they have no incoming edges
+ * Time Complexity: O(|E| + |V|), Space Complexity: O(n^2)
+ * Recommand Reading: http://cs.brown.edu/courses/csci0160/lectures/14.pdf
+ */
```
Comment (soapyigu) [inline DFS/CourseScheduleII.swift]: Leave an empty line before comments and class

**Comment by soapyigu** [comment]
Pretty awesome, just some styles need to be changed

File: DFS/CourseSchedule.swift (hunk 1 above)
Comment (demonkoo) [inline DFS/CourseSchedule.swift]: OK, done

File: DFS/CourseScheduleII.swift (hunk 3 above)
Comment (soapyigu) [inline DFS/CourseScheduleII.swift]: Thank you, merged. Could you please check the README as well? There are 3 parts need to be changed:

1. The number of questions we complete
2. DFS category
3. All questions list

Thank you, I will add you to the contribution list of README.

**Pull request merged**